xdem/coreg.py

```diff
@@ -2155,134 +2180,233 @@ def warp_dem(
     return warped.reshape(dem.shape)
 
 
-hmodes_dict = {
-    "nuth_kaab": NuthKaab(),
-    "nuth_kaab_block": BlockwiseCoreg(coreg=NuthKaab(), subdivision=16),
-    "icp": ICP(),
-}
+def create_inlier_mask(
+    src_dem: RasterType,
+    ref_dem: RasterType,
+    shp_list: list[str | gu.Vector | None] | tuple[str | gu.Vector] | tuple[()] = (),
+    inout: list[int] | tuple[int] | tuple[()] = (),
+    filtering: bool = True,
+    dh_max: AnyNumber = None,
+    nmad_factor: AnyNumber = 5,
+    slope_lim: list[AnyNumber] | tuple[AnyNumber, AnyNumber] = (0.1, 40),
+) -> NDArrayf:
+    """
+    Create a mask of inliers pixels to be used for coregistration. The following pixels can be excluded:
+    - pixels within polygons of file in shp_list (with corresponding inout element set to 1) - useful for masking \
+unstable terrain like glaciers.
+    - pixels outside polygons of file in shp_list (with corresponding inout element set to -1) - useful to \
+delineate a known stable area.
+    - pixels with absolute dh (=src-ref) are larger than a given threshold
```

Comment:
I first thought `src-ref` was a typo. Mind doing `(src_dem-ref_dem).abs()`  or `|src - ref|`, or `(src - ref).abs()`, whatever you prefer?

Reply:
Done in 94c6a07